tests/cxx/multiple_decl_namespace.cc

```diff
@@ -0,0 +1,55 @@
+//===--- multiple_decl_namespace.cc - test input file for iwyu ------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+// IWYU_ARGS: -I .
+
+// Tests that we handle multiply declared namespaces correctly.
+//
+// Where declarations of a namespace occur in multiple headers (either
+// because forward declarations were needed or otherwise), iwyu should
```

Comment:
I try to spell `iwyu` as `IWYU` in all new code/docs, so could you stick to that here and below?